demisto_sdk/commands/common/errors.py

```diff
@@ -367,12 +370,27 @@ def invalid_v2_integration_name():
     def found_hidden_param(parameter_name):
         return f"Parameter: \"{parameter_name}\" can't be hidden. Please remove this field."
 
+    @staticmethod
+    @error_code_decorator
+    def invalid_deprecated_integration(display_name, description):
+        error_answer = ''
+        if not display_name.startswith('Deprecated.'):
+            error_answer += 'The display_name (display) of your integration should start with "Deprecated.".\n'
```

Comment:
Let's make this change everywhere, to make it clear why the word deprecate should appear
```suggestion
            error_answer += 'The display_name (display) of all deprecated integrations should start with "Deprecated.".\n'
```